contrib/python-commentjson/template.py

```diff
@@ -0,0 +1,22 @@
+pkgname = "python-commentjson"
+pkgver = "0.9.0"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+]
+depends = ["python", "python-lark-parser"]
```

Comment:
```suggestion
depends = ["python-lark-parser"]
```

hard python is just for leafs